### ISSUE TITLE
SenselessMethodDuplicationInspector false positive

### ISSUE THREAD

**Issue by leofeyer** [kalessil/phpinspectionsea#1363]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v3.0.14 |
| Language level | PHP 7.1 |

```php
class A
{
    private $foo = 'foo';

    public function get(): string
    {
        return $this->foo;
    }
}

class B extends A
{
    private $foo = 'bar';

    public function get(): string
    {
        return $this->foo;
    }
}

$b = new B();
echo $b->get(); // bar
```

In this example, the `SenselessMethodDuplicationInspector` tells me that the `B::get()` method can be dropped as it is identical to the parent one. However, if I drop the method, it will return `A::$foo` instead of `B::$foo`:

```php
class A
{
    private $foo = 'foo';

    public function get(): string
    {
        return $this->foo;
    }
}

class B extends A
{
    private $foo = 'bar';
}

$b = new B();
echo $b->get(); // foo
```

So the method should not be dropped, even if it is identical to the parent one.

**Comment by kalessil** [comment]
Fixed!
